### ISSUE TITLE
I don't have permission to access the Notion Template

### ISSUE THREAD

**Issue by robinyu0910** [s4tlaw/RSS-reader-in-Notion#1]
sorry,  I have apply for the access to your Notion, could you please open the permission for me? Thank you!

**Comment by s4tlaw** [comment]
yeah i just published the website. in here https://mydock.notion.site/Guide-to-Build-How-to-make-a-RSS-reader-in-Notion-w-Python-e3b8f0ae5c864baab29ca31e5d340f64 .

**Comment by robinyu0910** [comment]
Thank you very much for your open-source project. The tutorials are very detailed and have helped me a lot!
